conf/test/titan/grakn-engine.properties

```diff
@@ -18,6 +18,7 @@
 
 #In-Memory or Distributed task manager
 taskmanager.implementation=ai.grakn.engine.tasks.manager.StandaloneTaskManager
+taskmanager.storage.zk=true
```

Comment:
All of these are `true` do we have a test in place which operates under the assumption of this being `false`. Just want to know if we tracking the support of the opposite of this config option.